tests/main/views/test_marketplace.py

```diff
@@ -740,15 +736,16 @@ def test_catalogue_of_briefs_page_filtered(self):
             "closed": False,
         }
 
-        parsed_original_url = urlparse(original_url)
-        parsed_prev_url = urlparse(document.xpath("//li[@class='previous']/a/@href")[0])
-        parsed_next_url = urlparse(document.xpath("//li[@class='next']/a/@href")[0])
-        assert parsed_original_url.path == parsed_prev_url.path == parsed_next_url.path
+        original_url = urlparse(original_url)
+        prev_url = urlparse(document.xpath("//li[@class='previous']/a/@href")[0])
+        next_url = urlparse(document.xpath("//li[@class='next']/a/@href")[0])
+        assert original_url.path == prev_url.path == next_url.path
 
-        normalize_qs = lambda qs: {k: set(v) for k, v in iteritems(parse_qs(qs)) if k != "page"}
```

Comment:
I don't think `exclude_page_key` gives the full story of why that function exists.  I'm pretty sure `normalize_qs` was introduced to deal with the issue of parameters not always being generated in the same order (maybe not the same order by Py2 and Py3), so tests would fail intermittently.  So while it does exclude the page key, it also fixes an additional issue.